node/src/command.rs

```diff
@@ -327,6 +327,26 @@ pub fn run() -> Result<()> {
 				}
 			})
 		},
+		#[cfg(feature = "try-runtime")]
+		Some(Subcommand::TryRuntime(cmd)) => {
+			let runner = cli.create_runner(cmd)?;
+			let chain_spec = &runner.config().chain_spec;
+			with_runtime_or_err!(chain_spec, {
+				{
+					// grab the task manager.
+					let registry =
+						&runner.config().prometheus_config.as_ref().map(|cfg| &cfg.registry);
+					let task_manager = sc_service::TaskManager::new(
```

Comment:
idk enough about this, but TaskManager seems to be for running async tasks. migrations wouldn't be running async, right? cause we'd probably want to run certain migrations synchronously (some less so, but for some, we'd probably want run migrations first before we start accepting extrinsics.)

Reply:
this is for running the node w/ try-runtime